nixos/modules/config/xdg/portal.nix

```diff
@@ -0,0 +1,38 @@
+{ config, pkgs ,lib ,... }:
+with lib;
+{
+  options.xdg.portal = {
+    enable =
+      mkEnableOption "<link xlink:href='https://github.com/flatpak/xdg-desktop-portal'>xdg desktop integration</link>"//{
+        default = true;
```

Comment:
I don't see why this should be default enabled, the other option wasn't

Reply:
The other XDG modules are on by default, though. And what *this* module does is make it possible for *other* modules to add portals. So the flatpak one still isn't doing anything by default. Otherwise all the other modules that add portals will also have to do `xdg.portal.enable=true`.

Reply:
True default would fix https://github.com/NixOS/nixpkgs/issues/53454

Reply:
Thanks for explaining the situation :+1: